queue/private.php

```diff
@@ -85,6 +85,10 @@
 							<button type="button" class="btn" class="paperworkComplete" ng-disabled="!client.checkedIn" ng-class="client.paperworkComplete ? 'btn-primary': 'btn-secondary' " ng-click="pwFilledOut()">Completed Paperwork</button>
 						</br>
 							<button type="button" class="btn" class="preparing" ng-disabled="!client.paperworkComplete" ng-class="client.preparing ? 'btn-primary': 'btn-secondary' " ng-click="nowPreparing()">Preparing</button>
+						</br>
+							<select ng-disabled="!client.preparing" ng-model="selectedVolunteer" ng-options="volunteer.name for volunteer in volunteers track by volunteer.userId">
+								<option value="">-- Select Preparer --</option>
```

Comment:
Do we want the "-- Select Preparer --" option to be selectable or should it have ```style="display:none"```?